.github/workflows/Python.yml

```diff
@@ -200,6 +200,9 @@ jobs:
         ls duckdb_tarball
         export DISTUTILS_C_COMPILER_LAUNCHER=ccache
         # TODO: Use ccache inside container, see https://github.com/pypa/cibuildwheel/issues/1030
+        if [[ "$GITHUB_REF" =~ ^refs/tags/v.+$ ]] ; then
+          export CIBW_TEST_COMMAND='python -m pytest {project}/tests/fast/api/test_dbapi00.py  --verbose'
```

Comment:
Not disabling these tests?

Reply:
Just set the command to `echo`? I think there's also a CIWB_SKIP environment variable, but I don't know how to use it off hand